samples/audioProfile.js

```diff
@@ -14,43 +14,38 @@
  */
 
 'use strict';
+function synthesizeSpeechAsync(request = {}) {
+  // Imports the Google Cloud client library
+  const textToSpeech = require('@google-cloud/text-to-speech');
+  // Creates a client
+  const client = new textToSpeech.TextToSpeechClient();
+  return new Promise((resolve, reject) => {
+    client.synthesizeSpeech(request, (err, response) => {
+      if (err) reject(err);
+      else resolve(response);
+    });
+  });
+}
 
-function synthesizeText(
+async function synthesizeText(
   text,
   outputFile,
   effectsProfileId,
   languageCode,
   ssmlGender
 ) {
   //[START tts_synthesize_text_audio_profile_beta]
-
-  // Imports the Google Cloud client library
-  const speech = require('@google-cloud/text-to-speech');
```

Comment:
We keep this stuff all around because they're used in snippets.  We want to keep all of the require statements and client creations.

Reply:
Done!